src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbblocklistblock.component.js

```diff
@@ -51,10 +51,17 @@
             $scope.valFormManager = model.valFormManager;
 
             if (model.stylesheet) {
+                var stylesheetUrl = model.stylesheet;
+                if (Umbraco.Sys.ServerVariables.application) {
+                    var rnd = Umbraco.Sys.ServerVariables.application.cacheBuster;
+                    var _op = (stylesheetUrl.indexOf("?") > 0) ? "&" : "?";
+                    stylesheetUrl = stylesheetUrl + _op + "umb__rnd=" + rnd;
+                }
```

Comment:
Instead of duplicating code, the following private function could be made public as e.g. `assetsService.appendCacheBuster(url)`:

https://github.com/umbraco/Umbraco-CMS/blob/cb47e0f1f6c4832ed4f1c90c4541c74adabea923/src/Umbraco.Web.UI.Client/src/common/services/assets.service.js#L48-L58

Reply:
Yes, I thought about this too, but I didn't want to touch more code than necessary to resolve the issue. I see now that this issue also apply to the custom view of the blocklistblock, so I think I will propose a new solution using the function from assetsService.